introduction_to_amazon_algorithms/image_classification_tensorflow/Amazon_TensorFlow_Image_Classification.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<p>I would reword the link without the number (i.e. <a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/#3.-Run-inference-on-the-pre-trained-model" target="_blank">Run inference on the pre-trained model</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>done.</p>

Reply:
<p>I see, if you really want the section numbering, definately putting the word "Section" helps to clarify what the 3 is referring to. In this case, I would put "Section 3 - <a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/#3.-Run-inference-on-the-pre-trained-model" target="_blank">Run inference on the pre-trained model</a>" as the whole link. Without the section numbering i.e. "<a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3579/#3.-Run-inference-on-the-pre-trained-model" target="_blank">Run inference on the pre-trained model</a>" is acceptable too, either way</p>

Reply:
<p>done. Thanks.</p>